docs/messaging/rabbitmq-integration.md

```diff
@@ -0,0 +1,309 @@
+    "RabbitMQ": {
+      "Client": {
+        "ConnectionString": "amqp://username:password@localhost:5672",
+        "DisableHealthChecks": true,
+        "DisableTracing": true,
+        "MaxConnectRetryCount": 2
+      }
+    }
+  }
+}
+```
+
+For the complete RabbitMQ client integration JSON schema, see [Aspire.RabbitMQ.Client/ConfigurationSchema.json](https://github.com/dotnet/aspire/blob/v8.2.1/src/Components/Aspire.RabbitMQ.Client/ConfigurationSchema.json).
+
+#### Use inline delegates
+
+Also you can pass the `Action<RabbitMQClientSettings> configureSettings` delegate to set up some or all the options inline, for example to disable health checks from code:
+
+```csharp
+builder.AddRabbitMQClient(
+    "messaging",
+    static settings => settings.DisableHealthChecks = true);
+```
+
+You can also set up the [IConnectionFactory](https://rabbitmq.github.io/rabbitmq-dotnet-client/api/RabbitMQ.Client.IConnectionFactory.html) using the `Action<IConnectionFactory> configureConnectionFactory` delegate parameter of the `AddRabbitMQClient` method. For example to set the client provided name for connections:
+
+```csharp
+builder.AddRabbitMQClient(
+    "messaging",
+    configureConnectionFactory:
+        static factory => factory.ClientProvidedName = "MyApp");
+```
+
+### Client integration health checks
+
+By default, .NET Aspire integrations enable [health checks](../fundamentals/health-checks.md) for all services. For more information, see [.NET Aspire integrations overview](../fundamentals/integrations-overview.md).
+
+The .NET Aspire RabbitMQ integration:
+
+- Adds the health check when <xref:Aspire.RabbitMQ.Client.RabbitMQClientSettings.DisableHealthChecks?displayProperty=nameWithType> is `true`, which attempts to connect to and create a channel on the RabbitMQ server.
+- Integrates with the `/health` HTTP endpoint, which specifies all registered health checks must pass for app to be considered ready to accept traffic.
+
+### Observability and telemetry
+
+.NET Aspire integrations automatically set up Logging, Tracing, and Metrics configurations, which are sometimes known as *the pillars of observability*. For more information about integration observability and telemetry, see [.NET Aspire integrations overview](../fundamentals/integrations-overview.md). Depending on the backing service, some integrations might only support some of these features. For example, some integrations support logging and tracing, but not metrics. Telemetry features can also be disabled using the techniques presented in the [Configuration](#configuration) section.
+
+#### Logging
+
+The .NET Aspire RabbitMQ integration uses the following log categories:
+
+- `RabbitMQ.Client`
+
+#### Tracing
+
+The .NET Aspire RabbitMQ integration emits the following tracing activities using OpenTelemetry:
+
+- `Aspire.RabbitMQ.Client`
+
+#### Metrics
+
+The .NET Aspire RabbitMQ integration currently doesn't support metrics by default. Check back for updates on this feature.
```

Comment:
The "check back" could be an empty promise, and we don't want promises in docs. You should remove that clause. I think it's pretty clear that it could be coming in the future. If you have a github issue to link to, you could link and introduce that as a way to watch progress on the feature.
```suggestion
The .NET Aspire RabbitMQ integration currently doesn't support metrics by default.
```